Gopkg.lock

```diff
@@ -56,10 +56,16 @@
     "envoy/api/v2/listener",
     "envoy/api/v2/route",
     "envoy/config/accesslog/v2",
+    "envoy/config/bootstrap/v2",
```

Comment:
These dependencies got added when I run `dep ensure` after updating `Gopkg.toml`. I double-checked if it is genuine by running `dep check` and looks like these were missing from `Gopkg.lock` earlier.

```
➜  contour git:(client-go-update-1.12) ✗ dep check
# Gopkg.lock is out of sync:
github.com/envoyproxy/go-control-plane/envoy/config/bootstrap/v2: imported or required, but missing from Gopkg.lock's input-imports
github.com/envoyproxy/go-control-plane/envoy/config/metrics/v2: imported or required, but missing from Gopkg.lock's input-imports
```

Reply:
Who can say, dep is pretty buggy